ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator.hs

```diff
@@ -6,6 +6,7 @@ module Ouroboros.Consensus.HardFork.Combinator (
   ) where
 
 -- Defines 'SingleEraBlock' and 'CanHardFork'
+-- Defines 'SingleEraInfo' and 'LedgerEraInfo'
```

Comment:
Join the two enumerations? Not that important

Reply:
Yeah, I thought about it, but they're kind of of a different nature, so I left it.